docker/linux/Dockerfile

```diff
@@ -52,6 +49,12 @@ RUN apt-get update && \
         wget \
         zstd \
     && \
+    if [ "${TARGETARCH}" = 'amd64' ]; then \
```

Comment:
Shouldn't we still install different GCC compilers on non amd64 as well? E.g. if one uses arm64 natively but wants to cross compile for amd64 instead.

Reply:
Yes, but I think I included this because I had issues ... but that is a lot of changes and bug-fixes ago, so I will test it again.

Reply:
Jup, seems to actually work fine. I will remove the `IF` and cleanup it up a bit.

I also noticed that I by accident included the `test_linux.sh` file, I will remove that for now.

Reply:
OK, this should be much better now.